code/game/machinery/alarm.dm

```diff
@@ -798,7 +790,9 @@
 			else if(wiresexposed && wires.IsAllCut() && iswirecutter(W))
 				buildstage = 1
 				update_icon()
-				new /obj/item/stack/cable_coil(get_turf(src),5)
+				user.visible_message("<span class='attack'>[user] has cut the wiring from \the [src]!</span>", "You have cut the last of the wiring from \the [src].")
+				playsound(get_turf(src), 'sound/items/Wirecutter.ogg', 50, 1)
+				new /obj/item/stack/cable_coil(get_turf(user),5)

```

Comment:
You should use pooling for this.
`getFromPool(/obj/item/stack/cable_coil, get_turf(user), 5)`
Cable coils are all pooled.